pkg/api/types.go

```diff
@@ -705,6 +713,8 @@ type ServiceSpec struct {
 	// not be changed by updates.
 	PortalIP string `json:"portalIP,omitempty"`
 
+	Rewrite bool `json:"rewrite,omitempty" description:"Does the load balancer rewrite packets or pass them through"`
```

Comment:
I'm not sure it's correct to put it in Status - it's part of the specification of the loadbalancer, it just happens to come from the implementation rather than the user.